h2o-algos/src/main/java/hex/deeplearning/DeepLearningModelInfo.java

```diff
@@ -205,7 +205,7 @@ public DeepLearningModelInfo(final DeepLearningParameters params, Key model_id,
         Log.warn(" *) Limit the size of the first hidden layer");
         if (dinfo._cats > 0) {
           Log.warn(" *) Limit the total number of one-hot encoded features with the parameter 'max_categorical_features'");
-          Log.warn(" *) Run h2o.interaction(...,pairwise=F) on high-cardinality categorical columns to limit the factor count, see http://learn.h2o.ai");
+          Log.warn(" *) Run h2o.interaction(...,pairwise=F) on high-cardinality categorical columns to limit the factor count, see http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/deep-learning.html#faq");
```

Comment:
`h2o.interaction` is the same for both python[1] and R. The only thing not right for python is the `pairwise=F` (`F` should be `False`) which I think is not a big deal and it keeps the message shorter


[1] https://github.com/h2oai/h2o-3/blob/03c73caf76c4eb4069e70490140435f0bc64f79d/h2o-py/h2o/frame.py#L4310

Reply:
Should we also mention the `max_categorical_features` here?

We could use the text from the FAQ:  We recommend either reducing the number of categorical factor levels upfront (e.g., using h2o.interaction()), or specifying `max_categorical_features` to use feature hashing to reduce the dimensionality.

Reply:
It is mentioned on the previous line (207) which is actually how I found out about it.